### DISCUSSION TITLE
Do pimcore have the functionality/options to disable localized tabs

### DISCUSSION THREAD

**Discussion by Karthi96** [pimcore/pimcore#9166] [Q&A / Support] [answered]
Hi Team, 

What I was looking at is, we have around 10 locales in the global setting. Under each class, we have created localized fields and it's displaying all locales.  Is there any config where we can specify how many locales to display in UI or to load only those locales on data objects opening?  

I'm looking to increase the performance as well as to control the locales on view?

any idea/suggestions would be welcome, thanks in advance.

**Comment by johanlindgren** [comment, marked as answer]
Look at the configuratuion options on the user, the role and the row in the workspace that assigns the permissions. On all these places you have configuration that can limit the amount of languages shown. So if the same logice applys to all objects consider setting this logic on the role, if th elogic depends on the  object path use the setting in the workspace found in the yellow config-button on the right side of all the checkboxes
